example/lib/widgets/modal_bottom_sheet.dart

```diff
@@ -239,7 +239,7 @@ Future<T> showModalBottomSheet<T>({
       context,
       _ModalBottomSheetRoute<T>(
         builder: builder,
-        theme: Theme.of(context, shadowThemeOnly: true),
+        theme: Theme.of(context),//, shadowThemeOnly: true),
```

Comment:
Is this something we need to do?

Reply:
Yeah, this parameter is no longer supported. As mentioned in this [link](https://github.com/flutter/flutter/issues/70981), we can now use CapturedThemes instead or Theme.of(context) without `shadowThemeOnly` parameter.